tests/golden/check-local-references-outside-repo/config-virtual-file.yaml

```diff
@@ -0,0 +1,26 @@
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+#
+# SPDX-License-Identifier: Unlicense
+traversal:
+  ignored: []
+verification:
+  anchorSimilarityThreshold: 0.5
+
+  externalRefCheckTimeout: 10s
+
+  notScanned: []
+
+  virtualFiles:
+    - ../link-target.md
+  ignoreRefs:
+    - ^(https?|ftps?)://(localhost|127\.0\.0\.1).*
+
+  ignoreAuthFailures: true
+
+  defaultRetryAfter: 30s
+
+  maxRetries: 3
+
+scanners:
+  markdown:
+    flavor: GitHub
```

Comment:
Now that #159 is merged, we could trim this file to simply this:

```md
verification:
  virtualFiles:
    - ../link-target.md
```

Next time you rebase on master, can you please trim this file?